src/libAtomVM/otp_crypto.c

```diff
@@ -561,6 +563,52 @@ static term nif_crypto_crypto_one_time(Context *ctx, int argc, term argv[])
     RAISE_ERROR(make_crypto_error(__FILE__, source_line, err_msg, ctx));
 }
 
+// not static since we are using it elsewhere to provide backward compatibility
+term nif_crypto_strong_rand_bytes(Context *ctx, int argc, term argv[])
+{
+    UNUSED(argc);
+
+    term count_term = argv[0];
+    VALIDATE_VALUE(count_term, term_is_integer);
+    avm_int_t out_len = term_to_int(count_term);
+    if (out_len < 0) {
+        RAISE_ERROR(BADARG_ATOM);
+    }
+
+    static bool initialized = false;
+    static mbedtls_entropy_context entropy_ctx;
+    static mbedtls_ctr_drbg_context rnd_ctx;
```

Comment:
So eventually this is not shared with ssl?

Reply:
This is still a draft, I  just wanted to have something working to get tests running, so we can iterate over this without (hopefully) introducing regressions.

Entropy and random context will be part of GenericUnixPlatform struct (same thing will apply also for other platforms using mbedtls).
Platform will initialize entropy with platform dependent entropy.